client/webserver/site/src/js/markets.js

```diff
@@ -104,6 +104,18 @@ export default class MarketsPage extends BasePage {
       swapBttns(page.limitBttn, page.marketBttn)
       this.setOrderVisibility()
     })
+    bind(page.rateField, 'wheel', () => {
+      page.rateField.blur()
+    })
+    bind(page.lotField, 'wheel', () => {
+      page.lotField.blur()
+    })
+    bind(page.qtyField, 'wheel', () => {
+      page.qtyField.blur()
+    })
+    bind(page.mktBuyField, 'wheel', () => {
+      page.mktBuyField.blur()
+    })
```

Comment:
I'm not sure that 'blur' is the ideal behavior. How about accepting the `Event` argument and doing a `Event.preventDefault()` instead?

Reply:
That works!